app/models/channel/driver/imap.rb

```diff
@@ -262,12 +357,15 @@ def already_imported?(message_id, message_meta, count, count_all, keep_on_server
 
     # verify if message is already imported via same channel, if not, import it again
     ticket = article.ticket
-    if ticket&.preferences && ticket.preferences[:channel_id].present? && channel.present?
-      return false if ticket.preferences[:channel_id] != channel[:id]
+    #if ticket&.preferences && ticket.preferences[:channel_id].present? && channel.present?
```

Comment:
I wasn't quite sure of the purpose of this check, i've restored it but it seems like its a false negative... if the message id already came in via on channel, its highly likely its the same email even if its via a new channel.